<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attendance System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="Content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_pages">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="login_page">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <pointsize>40</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Attendance System</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="2" column="1" alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="authenticate_login_btn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Log In</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Password:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Username:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="login_username_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="login_password_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QPushButton" name="signup_btn">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Sign Up</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="signup_page">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>40</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Sign Up</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="3" column="2">
                 <widget class="QLineEdit" name="signup_username_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Password:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Username:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLineEdit" name="signup_lastname_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2" alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="authenticate_signup_btn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Sign Up</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TC No:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Lastname:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLineEdit" name="signup_tc_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="signup_name_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLineEdit" name="signup_password_input">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QPushButton" name="login_btn">
                <property name="text">
                 <string>Log In</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
